data/yrncollo.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "YrnCollo",
+  "type": "personal",
+  "bio": "Open Source Contributor | Cyber Security researcher| Reverse engineer | Red teamer",
+
+  "tags": [
+    "Open Source",
+    "C++",
+    "Python",
+    "C",
+    "VimScript"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/Yrn_Collo" },
+    { "icon": "FaGithub", "url": "https://github.com/yrncollo" },
+    { "icon": "FaYoutube", "url": "https://youtube.com/yrncollo" }
+  ],
+  "links": [
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/Yrn_Collo",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Blog: Let's share ideas",
+      "url": "https://yrncollo.github.io/",
+      "icon": "FaMicroBlog",
```

Comment:
this should be FaMicroblog
https://react-icons.github.io/react-icons/search?q=microblog

Reply:
I have just updated it